liblxqt-config-cursor/crtheme.cpp

```diff
@@ -265,13 +275,33 @@ bool applyTheme(const XCursorThemeData &theme, int cursorSize)
     for (const QString &name : qAsConst(names))
     {
         Cursor cursor = (Cursor)theme.loadCursorHandle(name);
-        XFixesChangeCursorByName(QX11Info::display(), cursor, QFile::encodeName(name).constData());
+
+        if(QGuiApplication::platformName() == QStringLiteral("xcb"))
+            XFixesChangeCursorByName(QX11Info::display(), cursor, QFile::encodeName(name).constData());
```

Comment:
in `applyTheme()`, there are too many xcb checks (especially inside the loop).

If your goal is returning `true` from this function when the platform is not xcb, IMHO, it's better to do it once, before the comment `// Sets default cursor size`:

```c++
    if(QGuiApplication::platformName() != QStringLiteral("xcb"))
        return true;
```

Reply:
OK